lib/index.js

```diff
@@ -1,6 +1,6 @@
 /* @flow */
 
-import React, { Component } from 'react';
+import React from 'react';
```

Comment:
For `.mjs` files (which is generated by microbundle) webpack only allows strict CJS imports (with named exports).
Further reading: https://lifesaver.codes/answer/webpack-4-compatibility-issue

Reply:
Fixed by not using `*.mjs` file extension for **module** field, so this is reverted.